sonic-net/sonic-mgmt
# Enhancement to Order ECMP test case

What I did:
Enhance order ecmp test case with following changes:
    
1. Added Scenario to verify ECMP Acceleration path (ECMP Member removal and addition)
2. Increase flow scale to 50 Ipv4 flows (incremental source and destination TCP Port) to get better entropy
3. Validated across more ASIC like TD2/TH2
4. Added test plan for all the tests.

How I did:

To verify ECMP Acceleration: 
1. shut/no-shut peer fanout port to trigger Oper down Notification on dut so that OA will trigger ECMP Acceleration path.
2. shutdown BGP service. This simulate the scenario where we don't get any FRR update of Route Removal and only ECMP Member will be removed/added to the Group by the OA

How I verify:
Validated on different platforms running 202205 image.
